sql_generators/glean_usage/templates/events_first_seen_v1.query.sql

```diff
@@ -0,0 +1,131 @@
+{{ header }}
+
+{% raw %}
+{% if is_init() %}
```

Comment:
Wouldn't the init table require the same columns as the regular one? Like event_first_seen_date and submission_date?